app/main/views/marketplace.py

```diff
@@ -75,14 +75,25 @@ def terms_and_conditions():
 def get_brief_by_id(framework_framework, brief_id):
     briefs = data_api_client.get_brief(brief_id)
     brief = briefs.get('briefs')
+    brief_responses = data_api_client.find_brief_responses(
+        brief_id=brief_id,
+        status='draft,submitted'
+    ).get("briefResponses")
+    started_brief_responses_count = len([response for response in brief_responses if "submittedAt" not in response])
+    completed_brief_responses_count = len([response for response in brief_responses if "submittedAt" in response])
+
     if brief['status'] not in ['live', 'closed', 'withdrawn'] or brief['frameworkFramework'] != framework_framework:
         abort(404, "Opportunity '{}' can not be found".format(brief_id))
 
-    if getattr(current_user, "supplier_id", None) is None:
-        # user unauthenticated or not a supplier
-        brief_responses = None
-    else:
-        brief_responses = data_api_client.find_brief_responses(brief_id, current_user.supplier_id)["briefResponses"]
+    has_supplier_responded_to_brief = False
+    if brief_responses and getattr(
+        current_user,
+        "supplier_id",
+        None
+    ) and current_user.supplier_id in brief_responses[0].values():
```

Comment:
`brief_responses` is at this point a `list` of `dict` objects like so:
```
[
    {'supplierId': 1, 'briefId': 123, 'id': 1234 ...}, 
    {'supplierId': 2, 'briefId': 123, 'id': 1235 ...}, 
    {'supplierId': 3, 'briefId': 123, 'id': 1236 ...}
]``` 
That means that the check:
`current_user.supplier_id in brief_responses[0].values()`
will check if the supplier id is in the values of the first element of the list, ie:
`[1, 123, 1234]`
But we want the _all_ the supplier ids:
`[1, 2, 3]`